### PR TITLE
[homematic] Smaller bug fixes

### PR DESCRIPTION
**Pull request by MHerbst**
The PR contains some smaller bug fixes to the implementation of the binding and the documentatioan.

### CODE REVIEW

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java [hunk 1]
```diff
@@ -173,8 +173,14 @@ protected HmDatapoint assembleDatapoint(String name, String unit, String type, S
         }
 
         HmValueType valueType = HmValueType.parse(type);
-        if (valueType == null || valueType == HmValueType.UNKNOWN) {
```
Comment (martinvw) [inline bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: I would really prefer to always include the braces

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java [hunk 2]
```diff
@@ -173,8 +173,14 @@ protected HmDatapoint assembleDatapoint(String name, String unit, String type, S
...
+        if (valueType == null || valueType == HmValueType.UNKNOWN)
             throw new IOException("Unknown datapoint type: " + type);
+        else if (valueType == HmValueType.FLOAT && dp.getUnit() == null
+                && StringUtils.contains(dp.getName(), "TEMPERATURE")) {
```
Comment (martinvw) [inline bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: Can this be matched more exact than contains, what are possible values?

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java (hunk 1 above)
Comment (MHerbst) [reply bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: I did not want to touch existing coding, but I agree with you and will add braces around the throw.

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java (hunk 2 above)
Comment (MHerbst) [reply bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: There a names with TEMPERATURE at the end and also names with TEMPERATURE in the middle. All datapoints with possibly missing units have in common that they are defined as float and contain "TEMPERATURE". I could check the documentation whether it makes sense to limit it more.

**Comment by MHerbst** [comment]
> Some comments, I hope @gerrieg can also review this?

He has not been active here for a long time and does not care about issues. I do not think he will participate in the review.

**Comment by martinvw** [comment]
> > Some comments, I hope @gerrieg can also review this?
> 
> He has not been active here for a long time and does not care about issues. I do not think he will participate in the review.

I see that he indeed was not active for a long time, I assume he has other priorities currently. We will proceed then without his input.

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java (hunk 2 above)
Comment (martinvw) [reply bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: Would be appreciated :-)

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java (hunk 1 above)
Comment (MHerbst) [reply bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: done

File: bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java (hunk 2 above)
Comment (MHerbst) [reply bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/parser/CommonRpcParser.java]: Replaced by regex that is more restrictive :-)

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**